lib/src/soloud.dart

```diff
@@ -147,101 +106,24 @@ interface class SoLoud {
   ///   }
   /// }
   /// ```
-  ///
-  /// The future will complete immediately (synchronously) if the engine is
-  /// either already initialized (`true`),
-  /// or it had failed to initialize (`false`),
-  /// or it was already shut down (`false`),
-  /// or it is _being_ shut down (`false`),
-  /// or when there wasn't ever a call to [init] at all (`false`).
-  ///
-  /// If the engine is in the middle of initializing, the future will complete
-  /// when the initialization is done. It will be `true` if the initialization
-  /// was successful, and `false` if it failed. The future will never throw.
-  ///
-  /// It is _not_ needed to await this future after a call to [init].
-  /// The [init] method already returns a future, and it is the
-  /// same future that this getter returns.
-  ///
-  /// ```dart
-  /// final result = await SoLoud.instance.initialize();
-  /// await SoLoud.instance.initialized;  // NOT NEEDED
-  /// ```
-  ///
-  /// This getter ([initialized]) is useful when you want to check the status
-  /// of the engine from places in your code that _don't_ do the initialization.
-  /// For example, a widget down the widget tree.
-  ///
-  /// If you need a version of this that is synchronous,
-  /// or if you don't care that the engine might be initializing right now
-  /// and therefore ready in a moment,
-  /// use [isInitialized] instead.
+  @Deprecated('Use isInitialized instead!')
```

Comment:
Yes, `init()` could be sync but not now because the `Loader` class must still be initialized there. Before `initilized` was using the completers used in the `init()` method. Now those completers have been removed and there is no more need of `initilized`. To know if the player is initialized there is `isInitialized` (like before) which directly query the SoLoud lib with FFI.

Reply:
But isn't SoLoud (the Dart package) kind of "uninitialized" until `Loader` is ready? And there is no way, even in the future, that `Loader` could be ready synchronously. So I'd say we keep the `initialized` future. It doesn't need to wait for the audio isolate anymore, but it should wait for Loader to be ready.

Reply:
As discussed, please remove the deprecation, keep the stump code, and add `// TODO(filiph): reimplement` or something like that.